drivers/media/i2c/ov5647.c

```diff
@@ -392,7 +429,10 @@ static int ov5647_sensor_power(struct v4l2_subdev *sd, int on)
 			goto out;
 		}
 
-		ret = __sensor_init(sd);
+		/*
+		 * Ensure streaming off to make clock lane go into LP-11 state.
+		 */
+		ret = ov5647_stream_off(sd);
```

Comment:
So we've just powered up, not set PLLs or anything, and yet only mess with CSI2 pad registers. Seems a little odd.

Reply:
This looks to be what the original code was doing, so I don't think @davidplowman wanted to change the behaviour.

Reply:
Having seen a similar patch for IMX219, it seems to be Renesas that has to see LP-11 state on the CSI2 lines.

If this is effectively how it was, then fine. It just looks odd.